projects/packages/forms/src/contact-form/class-contact-form-field.php

```diff
@@ -1141,6 +1145,35 @@ private function maybe_override_type() {
 		return $type;
 	}
 
+	/**
+	 * Determines if a form field is valid.
+	 *
+	 * Add checks here to confirm if any given form field
+	 * is configured correctly and thus should be rendered
+	 * on the frontend.
+	 *
+	 * @param string $type - the field type.
+	 *
+	 * @return bool
+	 */
+	public function is_valid_field( $type ) {
+		// Check for valid radio field.
+		if ( $type === 'radio' ) {
+			$options           = (array) $this->get_attribute( 'options' );
+			$non_empty_options = array_filter(
+				$options,
+				function ( $option ) {
+					return $option !== '';
+				}
+			);
+			if ( count( $non_empty_options ) === 0 ) {
+				return false;
+			}
```

Comment:
Nit: I think this can be reduced to a one liner:
```suggestion
			return count( $non_empty_options ) > 0;
```

Reply:
Yep, like it better!